src/util/linux-config.js

```diff
@@ -0,0 +1,19 @@
+import merge from 'lodash/merge';
```

Comment:
Argh, this should be `import merge from lodash.merge`. Thanks @MarshallOfSound for pointing that out.

Reply:
Fixed import.